From 01b3337df3544cef42be5243a25ed4e005c80efa Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (Red Hat)" <rostedt@goodmis.org>
Date: Fri, 18 Sep 2015 15:29:18 -0400
Subject: [PATCH 072/134] trace-cmd: Always use /var/run and /var/lib

Looking at documentation on the Internet, it seems that even if a tool is
installed in /usr/local, it still should be using /var/run and /var/lib, as
those are not places that will get overrided by upgrades, but all versions
of trace-cmd should know about it.

I still keep it dynamic for installing in $HOME, as that may not be done by
a administrative user.

Signed-off-by: Steven Rostedt <rostedt@goodmis.org>
---
 Makefile | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/Makefile b/Makefile
index c058adb1e121..62cb25b6c1bd 100644
--- a/Makefile
+++ b/Makefile
@@ -53,12 +53,6 @@ export man_dir man_dir_SQ html_install html_install_SQ INSTALL
 export img_install img_install_SQ
 export DESTDIR DESTDIR_SQ
 
-ifeq ($(prefix),/usr)
-root = /
-else
-root = $(prefix)/
-endif
-
 ifeq ($(prefix),$(HOME))
 plugin_dir = $(HOME)/.trace-cmd/plugins
 python_dir = $(HOME)/.trace-cmd/python
@@ -70,7 +64,7 @@ PLUGIN_DIR = -DPLUGIN_DIR="$(plugin_dir)"
 PYTHON_DIR = -DPYTHON_DIR="$(python_dir)"
 PLUGIN_DIR_SQ = '$(subst ','\'',$(PLUGIN_DIR))'
 PYTHON_DIR_SQ = '$(subst ','\'',$(PYTHON_DIR))'
-var_dir = $(root)var
+var_dir = /var
 endif
 
 VAR_DIR = -DVAR_DIR="$(var_dir)"
-- 
2.5.5

